src/commands/migrate.ts

```diff
@@ -33,6 +33,13 @@ export async function _migrate(
           parsedSettings,
           lastMigration,
         );
+        if (remainingMigrations.length > 0 || forceActions) {
```

Comment:
```suggestion
        const shouldExecuteActions = remainingMigrations.length > 0 || forceActions;
        if (shouldExecuteActions) {
```

Reply:
Please update `if (remainingMigrations.length > 0 || forceActions)` for `afterAllMigrations` below to be `if (shouldExecuteActions)`. Better to have this shared logic in one place.